cob_collision_velocity_filter/src/cob_collision_velocity_filter.cpp

```diff
@@ -52,62 +52,54 @@
 #include <visualization_msgs/Marker.h>
 
 // Constructor
-CollisionVelocityFilter::CollisionVelocityFilter()
+CollisionVelocityFilter::CollisionVelocityFilter(costmap_2d::Costmap2DROS * costmap)
 {
   // create node handle
   nh_ = ros::NodeHandle("~");
 
   m_mutex = PTHREAD_MUTEX_INITIALIZER;
 
-  // node handle to get footprint from parameter server
-  std::string costmap_parameter_source;
-  if(!nh_.hasParam("costmap_parameter_source")) ROS_WARN("Checking default source [/local_costmap_node/costmap] for costmap parameters");
-  nh_.param("costmap_parameter_source",costmap_parameter_source, std::string("/local_costmap_node/costmap"));
-
-  ros::NodeHandle local_costmap_nh_(costmap_parameter_source);
+  anti_collision_costmap_ = costmap;
 
   nh_.param("costmap_obstacle_treshold", costmap_obstacle_treshold_, 50);
 
   // implementation of topics to publish (command for base and list of relevant obstacles)
   topic_pub_command_ = nh_.advertise<geometry_msgs::Twist>("command", 1);
   topic_pub_relevant_obstacles_ = nh_.advertise<nav_msgs::OccupancyGrid>("relevant_obstacles_grid", 1);
 
+  std::string teleop_topic;
+  if(!nh_.hasParam("teleop_topic")) ROS_WARN("Used default parameter for teleop_topic [teleop_twist]");
+  nh_.param("teleop_topic",teleop_topic, std::string("teleop_twist"));
```

Comment:
we don't set topics by parameters.
Please remove.

This should be done via remappings in the respective launch files.